pipelines/deployment-pipelines/wait-for-workloads-readiness-or-fail/pipeline.yaml

```diff
@@ -0,0 +1,47 @@
+name: Wait for Workloads readiness or fail
+
+# This pipeline is a deployment request pipeline and runs for any deploy or re-deploy.
+on:
+  deployment_request: {}
+
+# This pipeline needs permissions to deploy to our development environments. We could add staging and production here too.
+permissions:
+  application: developer
+  env-types:
+    development: deployer
+
+# Because this pipeline runs tests, we're going to use concurrency controls to ensure only one deploy and test cycle is running at a time against the target environment.
+concurrency:
+  group: "${{ pipeline.id }}-${{ pipeline.app.id }}-${{ inputs.env_id }}"
+
+jobs:
+  deploy:
+    steps:
+    - if: ${{ ! inputs.set_id }}

```

Comment:
probably useful to indicate that these two steps are copied from teh default deployment pipeline